aws/lambda/index.js

```diff
@@ -65,16 +65,23 @@ exports.handler = async function(event, context) {
             QueueUrl: process.env["QUEUE_URL"],
             MessageGroupId: "0",
             MessageAttributes: {
-                "pr": StringValue(pull_request["number"]),
-                "base": StringValue(pull_request["base"]["ref"]),
+                "base-branch": StringValue(pull_request["base"]["ref"]),
                 "base-sha": StringValue(pull_request["base"]["sha"]),
-                "head": StringValue(pull_request["head"]["sha"]),
                 "head-branch": StringValue(pull_request["head"]["ref"]),
-                "repo": StringValue(json["repository"]["clone_url"]),
+                "head-sha": StringValue(pull_request["head"]["sha"]),
                 "installation": StringValue(json["installation"]["id"]),
+                "pr": StringValue(pull_request["number"]),
                 "url": StringValue(pull_request["base"]["repo"]["url"])
             }
         };
+        if ("clone_url" in pull_request["base"]["repo"]) {
+            message["MessageAttributes"]["base-repo"] = StringValue(pull_request["base"]["repo"]["clone_url"]);
+            message["MessageAttributes"]["head-repo"] = StringValue(pull_request["head"]["repo"]["clone_url"]);
```

Comment:
It seems that if `head` is not `jekyll/jekyll`, then Bundler cannot find the commit when running on EC2.

I cannot reproduce this issue locally using the same version of Bundler. I am running a newer version of Git locally, which may contribute?